diff --git a/CMakeLists.txt b/CMakeLists.txt
index 67bcb9c..8e8ab67 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -107,12 +107,12 @@ if(MSVC)
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /DYNAMICBASE")
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /guard:cf")
     # enable Spectre Mitigation, not supported by clang-cl
-    if((NOT CMAKE_CXX_COMPILER_ID STREQUAL Clang) AND (NOT CMAKE_CXX_COMPILER_ID STREQUAL IntelLLVM))
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Qspectre")
-    endif()
-    if((NOT CMAKE_C_COMPILER_ID STREQUAL Clang) AND NOT (CMAKE_C_COMPILER_ID STREQUAL IntelLLVM))
-        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /Qspectre")
-    endif()
+#    if((NOT CMAKE_CXX_COMPILER_ID STREQUAL Clang) AND (NOT CMAKE_CXX_COMPILER_ID STREQUAL IntelLLVM))
+#        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Qspectre")
+#    endif()
+#    if((NOT CMAKE_C_COMPILER_ID STREQUAL Clang) AND NOT (CMAKE_C_COMPILER_ID STREQUAL IntelLLVM))
+#        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /Qspectre")
+#    endif()
 endif()
 
 #CXX compiler support
